src/main/resources/scripts/vanilla gui.sk

```diff
@@ -0,0 +1,16 @@
+#
+# A vanilla example of a gui creation using Skript only
+# Requires Skript 2.3+
+#
+
+command /gui:
+	trigger:
+		set {_gui} to a new chest inventory with 6 row with name "Gui Test"
+		set slot 0 of {_gui} to stone
+		open {_gui} to player
+ 
+on inventory click:
+	if name of event-inventory is "Gui Test":
+		if index of event-slot = 0:
+			cancel event
+			send "You clicked on the slot 0 !"
```

Comment:
Should be using the message effect, not send, for consistency sake.

Reply:
Both are the same. We should let 2 different things to show to any user that oth possibilities work.

Reply:
New skript users may not know the difference, if you want to use send, would be nice to add a comment that the send effect is identical to the message effect

Reply:
I think not it will be useful to add that